src/cpp/session/modules/SessionRAddins.cpp

```diff
@@ -145,17 +149,19 @@ class AddinRegistry : boost::noncopyable
          BOOST_FOREACH(const std::string& key, addinsJson | boost::adaptors::map_keys)
          {
             json::Value valueJson = addinsJson.at(key);
-            if(json::isType<json::Object>(valueJson))
+            if (json::isType<json::Object>(valueJson))
             {
                bool interactive;
                std::string name, package, title, description, binding;
+               int ordinal = 0;
                Error error = json::readObject(valueJson.get_obj(),
                                               "name", &name,
                                               "package", &package,
                                               "title", &title,
                                               "description", &description,
                                               "interactive", &interactive,
-                                              "binding", &binding);
+                                              "binding", &binding,
+                                              "ordinal", &ordinal);
```

Comment:
Will this cause errors when RStudio attempts to read addin registries persisted by earlier versions?

Reply:
Good catch! I've updated the PR to be a bit more defensive here.